imagestreams/ruby-centos7.json

```diff
@@ -49,14 +55,17 @@
           "openshift.io/provider-display-name": "Red Hat, Inc.",
           "description": "Build and run Ruby 2.2 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/tree/master/2.2/README.md.",
           "iconClass": "icon-ruby",
-          "tags": "builder,ruby",
+          "tags": "hidden,builder,ruby",
```

Comment:
Also, why add `hidden`, now, when `rh-ruby22` is still supported [till April 2018](https://access.redhat.com/support/policy/updates/rhscl).

IMHO we should push those changes in April, not in March.

Reply:
these changes are going to ship with OCP3.10 which will ship post april 2018.

it does mean we've "prematurely" marked the centos imagestreams as hidden, but for consistency this was easier and given that we've got newer images available that people should be using anyway, i wasn't going to lose a lot of sleep over it.

Reply:
Sure, no worries.

I just thought there should be all-in-one process (from our side) of EOLing images. Instead of squeezing this tiny change past almost unnoticed, It'd be IMHO better if you created an issue, so we could solve this properly.

Reply:
Fair enough, in the future we can definitely do it that way, and I agree that's better.  Was just trying to speed things along since I was touching all the files anywa.